src/dirs.proj

```diff
@@ -58,7 +58,11 @@
     </TraversalBuildDependsOn>
   </PropertyGroup>
 
-
-
+  <!-- Workaround to remove items from PackagesNuSpecFiles (defined in packages.targets) -->
```

Comment:
Maybe also comment that they're added via wildcard there, so there's not a great way to remove them otherwise?